src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import static java.lang.Integer.parseInt;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    public static final DateTimeFormatter DDMMYYYY = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
+    public static boolean between(LocalDate d, LocalDate from, LocalDate to) {
+        return (d.isAfter(from) || d.isEqual(from)) && (d.isBefore(to) || d.isEqual(to));
+    }
```

Comment:
All methods that we use only in the class must be private
All private methods should be below public methods
do not use such naming(`d`), how to guess what it stores?
```suggestion
    private boolean between(LocalDate d, LocalDate from, LocalDate to) {
        return (d.isAfter(from) || d.isEqual(from)) && (d.isBefore(to) || d.isEqual(to));
    }
```